.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,68 @@
+name: '🐞 Bug report'
+description: Let us know about functionality that is not working as expected
```

Comment:
>Let us know about functionality  [](http://example.com/codeflow?start=13&length=32)

How about 'report that something is not working as expected'?

Reply:
Done 👍